test/inductor/test_cpu_cpp_wrapper.py

```diff
@@ -255,7 +255,8 @@ class BaseTest(NamedTuple):
         BaseTest("test_multihead_attention", "cpu", test_cpu_repro.CPUReproTests()),
         BaseTest(
             "test_multi_threading",
-            code_string_count={"py::gil_scoped_release release;": 1},
+            # Two threads compile, so we expect the output code to be printed twice.
```

Comment:
We weren't capturing the output from the cache hit path so I think this was passing by mistake